src/app/component/comments/components/comment-counter/comment-counter.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="counter">
+  <p *ngIf="[totalElements] <= 1">{{ totalElements }} comment</p>
+  <p *ngIf="[totalElements] > 1">{{ totalElements }} comments</p>
```

Comment:
Angular has own pattern to do it - "Showing an alternative template using else"

Reply:
But, as can I see - you don't need to use any <code>*ngIf</code>, because the different is in ending of the word - in some data, not in template. So, it can be used something like:
```js
comment{{totalElements>1 ? 's' : ''}}
```

Reply:
Fixed